readthedocs/projects/tasks/builds.py

```diff
@@ -626,6 +630,17 @@ def get_valid_artifact_types(self):
                         },
                     )
 
+                # Rename file as "<project_slug>-<version_slug>.<artifact_type>",
+                # which is the filename that Proxito serves for offline formats.
+                filename = list_dir[0]
+                _, extension = filename.rsplit(".")
+                path = Path(artifact_directory) / filename
+                destination = (
+                    Path(artifact_directory) / f"{self.data.project.slug}.{extension}"
+                )
+                assert_path_is_inside_docroot(path)
```

Comment:
We should check both paths

```suggestion
                assert_path_is_inside_docroot(path)
                assert_path_is_inside_docroot(destination)
```